### PR TITLE
Clean up passing properties to components

### PR DESCRIPTION
**Pull request by michaelrkn**

### CODE REVIEW

File: source/components/passing-properties-to-a-component.md [hunk 1]
```diff
@@ -1,97 +1,77 @@
-By default a component does not have access to any properties normally
-available in the template in which it is used (sometimes referred to
-as the template `scope`)
+Components only have access to properties passed to them.
```
Comment (locks) [inline source/components/passing-properties-to-a-component.md]: but what you pass in, aren't they attributes? :D

File: source/components/passing-properties-to-a-component.md [hunk 2]
```diff
@@ -1,97 +1,77 @@
...
 It is important to note that these properties stay in sync (technically
 known as being "bound"). That is, if the value of `componentProperty`
 changes in the component, `outerProperty` will be updated to reflect that
 change. The reverse is true as well.
 
-You can also bind properties from inside an `{{#each}}` loop. This will
-create a component for each item and bind it to each model in the loop.
+## Positional Params
+
+Instead of passing parameters in by name, you can pass them in by position.
```
Comment (locks) [inline source/components/passing-properties-to-a-component.md]: you can also use both of them at the same time, this sentence kinda makes it feel XOR

File: source/components/passing-properties-to-a-component.md [hunk 3]
```diff
@@ -1,97 +1,77 @@
...
 ```
-If you are using the `{{component}}` helper to render your component, you can
-pass properties to the chosen component in the same manner:
-
-```handlebars
-{{component componentName title=model.title name=model.name}}
-```
 
-### Positional Params
+To set the component up to recieve parameters this way, you need
```
Comment (locks) [inline source/components/passing-properties-to-a-component.md]: receive (you need a text expansion for this one)

File: source/components/passing-properties-to-a-component.md [hunk 4]
```diff
@@ -1,97 +1,77 @@
...
-Apart from passing attributes to a component, you can also pass in positional parameters
-by setting the `positionalParams` attribute in your component class.
-
-```app/components/x-visit.js
-const MyComponent = Ember.Component.extend();
-
-MyComponent.reopenClass({
-  positionalParams: ['name', 'model']
+```app/components/blog-post.js
+export default Ember.Component.extend({
```
Comment (locks) [inline source/components/passing-properties-to-a-component.md]: AFAIK this is the old way, which is the wrong way.

**Comment by locks** [comment]
I feel like some valuable information is lost in your edit of positional arguments.

File: source/components/passing-properties-to-a-component.md (hunk 3 above)
Comment (michaelrkn) [inline source/components/passing-properties-to-a-component.md]: i just need to learn how to spell ;)

File: source/components/passing-properties-to-a-component.md [hunk 5]
```diff
@@ -1,97 +1,88 @@
-By default a component does not have access to any properties normally
-available in the template in which it is used (sometimes referred to
-as the template `scope`)
+Components only have access to attributes passed to them.
```
Comment (locks) [inline source/components/passing-properties-to-a-component.md]: I like terse, but maybe expand a tiny bit? Remember you removed the whole index.

```
Components are isolated from their surroundings, so any data that the component needs has to be passed in.
```

?

**Comment by michaelrkn** [comment]
All done?!

File: source/components/passing-properties-to-a-component.md [hunk 6]
```diff
@@ -1,97 +1,89 @@
...
+BlogPostComponent.reopenClass({
+  positionalParams: ['title', 'body']
 });
 
-export default MyComponent;
+export default BlogPostComponent;
 ```
 
-This will expose the first parameter in your template as the `{{name}}` attribute and the second as `{{model}}`. They will also be available as regular attributes in your component via `this.get('name')` and `this.get('model')`.
+The you can use the attributes in the component exactly as if they had been
```
Comment (locks) [inline source/components/passing-properties-to-a-component.md]: you accidentally something at the beginning of the sentence

**Comment by michaelrkn** [comment]
Ok how about NOW?

**Comment by locks** [comment]
Not even waiting for CI. Que sera, sera.

**Comment by michaelrkn** [comment]
:boom:

**Pull request merged**